test/helper/helper.go

```diff
@@ -117,7 +117,12 @@ func TestRoot() *crdt.Root {
 // TextChangeContext returns the context of test change.
 func TextChangeContext(root *crdt.Root) *change.Context {
 	return change.NewContext(
-		change.InitialID,
+		change.NewID(
+			change.InitialClientSeq,
+			change.InitialServerSeq,
+			change.InitialLamport,
+			time.InitialActorID,
+			time.NewVersionVector()),
```

Comment:
@coderabbitai What if we remove the `var InitialID` and replace it with a function like `GetInitialID`? Since we shouldn't use `InitialID` as a value directly, I think creating a function would be a better approach.